app/router/router.tsx

```diff
@@ -19,6 +19,11 @@ export const COMMIT_PARAM_NAME = "commit";
 export const HOST_PARAM_NAME = "host";
 export const COMMAND_PARAM_NAME = "command";
 
+// Sort params for the global filter.
+
+export const SORT_BY_NAME = "sort-by";
+export const SORT_ORDER_NAME = "sort-order";
```

Comment:
(to match the consts above)

```suggestion
export const SORT_BY_PARAM_NAME = "sort-by";
export const SORT_ORDER_PARAM_NAME = "sort-order";
```